cypress/e2e/awx/infrastructure/instances/instances.cy.ts

```diff
@@ -4,7 +4,7 @@ import { Instance } from '../../../../../frontend/awx/interfaces/Instance';
 import { cyLabel } from '../../../../support/cyLabel';
 import { awxAPI } from '../../../../support/formatApiPathForAwx';
 
-cyLabel(['upstream'], () => {
+cyLabel(['openshift'], () => {
```

Comment:
I am adding a different solution for this one here, the API returns the deployment type thus we can verify if the API supports certain actions then skip the tests.

https://github.com/ansible/ansible-ui/pull/2716